commands/serve/web/components/property-panel/DataCube.jsx

```diff
@@ -27,25 +28,37 @@ const getValue = (data, reference, defaultValue) => {
 };
 
 export default function DataCube({ setProperties, target, properties }) {
-  const createHandler = target.propertyPath.match('/qHyperCube') ? hcHandler : loHandler;
+  // Store dimensions outside of handler to support filterpanes or other containers
+  // holding list objects, which do not store dimensions in an array (like hc.qDimensions).
+  const [dimensions, setDimensions] = useState([]);
```

Comment:
I don't see how this work on existing objects that already have dimensions?

Reply:
Yeah, it won't. It's not a huge limitation though since this is a dev tool and sufficient for continuing the Filter pane work.
I'm also not sure if we can deduce this in any way from the arguments we have here. In that case that would be a more reliable source for extracting dimensions.